src/main/java/tests/OperationHandlersTest.java

```diff
@@ -0,0 +1,65 @@
+package tests;
+
+import core.basesyntax.*;
```

Comment:
Avoid using wildcard imports. They can lead to unintended conflicts and make the code less readable since it's not clear which classes are used from the import.